include/gz/rendering/PixelFormat.hh

```diff
@@ -43,10 +43,10 @@ namespace gz
       PF_BAYER_RGGB8  = 4,
       /// < Bayer BGGR, 1-byte per channel
       PF_BAYER_BGGR8  = 5,
-      /// < Bayer GBGR, 1-byte per channel
-      PF_BAYER_GBGR8  = 6,
-      /// < Bayer GRGB, 1-byte per channel
-      PF_BAYER_GRGB8  = 7,
+      /// < Bayer GBRG, 1-byte per channel
+      PF_BAYER_GBRG8  = 6,
+      /// < Bayer GRBG, 1-byte per channel
+      PF_BAYER_GRBG8  = 7,
```

Comment:
Hey @iche033, I have made the changes to the enum elements corresponding to bayer formats so that they are consistent with gazebo-classic. As you suggested, we could now review the outcomes of the ABI build.

Reply:
Update: The ABI compatibility failed :(

Reply:
yes but looking closely at the [report](https://build.osrfoundation.org/job/ignition_rendering-abichecker-any_to_any-ubuntu_auto-amd64/1377/API_5fABI_20report/), the compatibility is 100%. It did catch the behavior change, which I'm leaning towards letting it slide since Bayer format was not working before.